starlite/parsers.py

```diff
@@ -1,6 +1,6 @@
 from contextlib import suppress
 from functools import reduce
-from typing import TYPE_CHECKING, Any, Dict, List, Tuple, cast
+from typing import TYPE_CHECKING, Any, Dict, List, Tuple, Union, cast
```

Comment:
The lgtm tool is complaining about this unused import, but it is required for the cast on line 46 (otherwise flake8 complains)